vm/systemSmartContracts/proto/governance.proto

```diff
@@ -7,19 +7,24 @@ option (gogoproto.stable_marshaler_all) = true;
 
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 
+enum VoteValueType {
+    Yes  = 0;
+    No   = 1;
+    Veto = 2;
+}
+
 message GeneralProposal {
     bytes  IssuerAddress  = 1 [(gogoproto.jsontag) = "IssuerAddress"];
-    bytes  GitHubCommit   = 2 [(gogoproto.jsontag) = "GitHubCommit"];
+    bytes  CommitHash     = 2 [(gogoproto.jsontag) = "CommitHash"];
     uint64 StartVoteNonce = 3 [(gogoproto.jsontag) = "StartVoteNonce"];
     uint64 EndVoteNonce   = 4 [(gogoproto.jsontag) = "EndVoteNonce"];
-    int32  Yes            = 5 [(gogoproto.jsontag) = "Yes"];
-    int32  No             = 6 [(gogoproto.jsontag) = "No"];
-    int32  Veto           = 7 [(gogoproto.jsontag) = "Veto"];
-    int32  DontCare       = 8 [(gogoproto.jsontag) = "DontCare"];
-    bool   Voted          = 9 [(gogoproto.jsontag) = "Voted"];
-    repeated bytes Voters = 10 [(gogoproto.jsontag) = "Voters"];
-    bytes  TopReference   = 11 [(gogoproto.jsontag) = "TopReference"];
-    bool   Closed         = 12 [(gogoproto.jsontag) = "Closed"];
+    bytes  Yes            = 5 [(gogoproto.jsontag) = "Yes", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
+    bytes  No             = 6 [(gogoproto.jsontag) = "No", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
+    bytes  Veto           = 7 [(gogoproto.jsontag) = "Veto", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
+    bool   Voted          = 8 [(gogoproto.jsontag) = "Voted"];
```

Comment:
I thought Voted is equal to Closed... I think a more appropriate name would be "Passed" or "Approved"

Reply:
renamed

Reply:
ok